smb/src/main/java/ch/cyberduck/core/smb/SMBProtocol.java

```diff
@@ -0,0 +1,85 @@
+package ch.cyberduck.core.smb;
+
+/*
+ * Copyright (c) 2002-2023 iterate GmbH. All rights reserved.
+ * https://cyberduck.io/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+import ch.cyberduck.core.AbstractProtocol;
+import ch.cyberduck.core.Credentials;
+import ch.cyberduck.core.LocaleFactory;
+import ch.cyberduck.core.LoginOptions;
+import ch.cyberduck.core.Scheme;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class SMBProtocol extends AbstractProtocol {
```

Comment:
Override `getUsernamePlaceholder` to provide placeholder text with username format requiring share name.